index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles/styles.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins&display=swap"
+      rel="stylesheet"
+    />
+    <title>Salman Ahmad Khan's Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <nav class="nav-bar">
+        <a class="logo" href="#home">My Logo</a>
+        <i class="fa-sharp fa-solid fa-bars"></i>
+      </nav>
+      <!-- write these commented elements for later use -->
+      <!--   <div>
+            <ul>
+                <li><a href="#">Menu</a></li>
+                <li><a href="#"></a>Contact Us</li>
+                <li><a href="#"></a>About</li>
+            </ul>
+        </div> -->
+    </header>
+    <main>
+      <section class="intropage">
+        <div class="intro">
+          <h1>Hey There. I'm Salman</h1>
+          <h2>I'm a Software Developer</h2>
+          <p>
+            I can help you to build a product, feature or website Look through
+            some of my work and experience! If you like what you see and have a
+            project you need coded, don't hesitate to contact me.
+          </p>
+        </div>
+        <!-- 
+        use section2 for social elements -->
+        <div class="sociallinks">
+          <img class="social" src="/img/Disabled.png" alt="" />
+          <img class="social" src="/img/Disabled-1.png" alt="" />
+          <img class="social" src="/img/Enabled.png" alt="" />
+          <img class="social" src="/img/Disabled-2.png" alt="" />
+          <img class="social" src="/img/Disabled-3.png" alt="" />
+        </div>
```

Comment:
- [ ] I would like to reiterate what the first reviewer highlighted. When listing items on a webpage, it is semantic to use `ul` and `li` tags or `ol` and `li` tags if you want the items ordered. During the listing, kindly wrap the img tags in a tags. So the structure would be like this:

```HTML
<ul>
  <li>
    <a href="mailto:user@eample.com"> <img class="social" src="/img/Disabled.png" alt="" /></a>
  </li>
  <li>
    <a href="https://github.com/username"><img class="social" src="/img/Enabled.png" alt="" /></a>
  </li>
  ...
</ul>
```